frontend/controller/router.js

```diff
@@ -252,7 +252,7 @@ var router = new Router({
       meta: {
         title: 'Join a Group'
       },
-      beforeEnter: createEnterGuards(loginGuard, mailGuard)
+      beforeEnter: createEnterGuards(loginGuard)
```

Comment:
If we will continue to have a mailGuard in the future, maybe it would be better to edit the guard itself  (returning always true) than removing it from where it's needed, so it's easier to add it back again.

Reply:
Good idea...

Reply:
Will resolve this by simply commenting out that line and the guard both (since it's no longer used, and unused vars is a linter error)